.github/workflows/package_for_test.yml

```diff
@@ -62,8 +67,9 @@ jobs:
           CI: false
 
       - name: Package for MacOS
-        if: matrix.os == 'macos-latest'
+        if: ${{ matrix.os == 'macos-latest' && env.APPLE_ID != '' }}
         run: |
+          export CSC_FOR_PULL_REQUEST=true
```

Comment:
what does this `CSC_FOR_PULL_REQUEST` do, this keyword is not found in https://www.electron.build/, and there's a warning about it at https://github.com/electron-userland/electron-builder/blob/e3fc9b544cc8c6728ffd77a45408d6e0e87dbb46/packages/app-builder-lib/src/codeSign/macCodeSign.ts#L39

Reply:
>  Current build is a part of pull request, code signing will be skipped.
Set env CSC_FOR_PULL_REQUEST to true to force code signing.

For electron builder, it will always skip code sign if the action is triggered by `pull_request` unless set CSC_FOR_PULL_REQUEST true.

Reply:
I'm not sure if we should take the risk mentioned at https://github.com/electron-userland/electron-builder/blob/e3fc9b544cc8c6728ffd77a45408d6e0e87dbb46/packages/app-builder-lib/src/codeSign/macCodeSign.ts#L39 to force PR triggering for packaging.

> "There are serious security concerns with CSC_FOR_PULL_REQUEST=true (see the  CircleCI documentation (https://circleci.com/docs/1.0/fork-pr-builds/) for details)" +
    "\nIf you have SSH keys, sensitive env vars or AWS credentials stored in your project settings and untrusted forks can make pull requests against your repo, then this option isn't for you."

It would be better to add this annotation next to the configuration.

Besides, what dost it cost if the package of a PR is not signed

Reply:
Maybe I can remove `CSC_FOR_PULL_REQUEST`, because for `pull_request` event, it cannot access other secret variables, so if `APPLE_ID ` is not null, it must be triggered by the `pull` event.